orchagent/portsorch.cpp

```diff
@@ -3609,17 +3701,24 @@ bool PortsOrch::removeVlan(Port vlan)
 
     sai_status_t status = sai_vlan_api->remove_vlan(vlan.m_vlan_info.vlan_oid);
     if (status != SAI_STATUS_SUCCESS)
+    /* Do not delete default VLAN from driver, but clear internal state */
```

Comment:
Please fix alignment and better add {} as the nested if branch is not the single lined one